rust/main/agents/validator/src/validator.rs

```diff
@@ -359,6 +365,10 @@ impl Validator {
                         ?announcement_location,
                         "Validator has announced signature storage location"
                     );
+
+                    // Set announced in metrics
+                    self.announced.set(1);
```

Comment:
It looks like it is the only place where we set this metric. If validator itself announces its location, this metric won't be sent until we restart the validator. I wonder if it is the expected behaviour?